arch/sparc/Kconfig

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2019-2020 Cobham Gaisler AB
+# SPDX-License-Identifier: Apache-2.0
+
+menu "SPARC Options"
+	depends on SPARC
+
+config ARCH
+	default "sparc"
+
+config SPARC_NWIN
+	int "Number of register windows"
+	default 8
+	help
+	  Number of implemented register windows.
+
+config GEN_ISR_TABLES
+	default y
+
+config GEN_IRQ_VECTOR_TABLE
+	default n
+
+config GEN_SW_ISR_TABLE
+	default y
+
+config NUM_IRQS
+	int
+	default 32
+
+config SPARC_CASA
+	bool "CASA instructions"
+	help
+	  Use CASA atomic instructions. Defined by SPARC V9 and available
+	  in some LEON processors.
+
+# The SPARC V8 ABI allocates a stack frame of minimum 96 byte for each SAVE
```

Comment:
these are some very large stack sizes!

Reply:
Yes, stacks are large due to the SPARC ABI stack frames. The stack Kconfig options have been revisited and most of them are reduced in the 2020-11-12 force push.

For example:
```
 config ISR_STACK_SIZE
-       default 8192
+       default 4096

 config TEST_EXTRA_STACKSIZE
-       default 8192 if COVERAGE_GCOV
-       default 4096
+       default 4096 if COVERAGE_GCOV
+       default 2048

 config SYSTEM_WORKQUEUE_STACK_SIZE
-       default 8192
+       default 4096
```